host/3.0/buster/amd64/functions-base/functions-host.Dockerfile

```diff
@@ -0,0 +1,31 @@
+ARG HOST_VERSION=3.0.15584
+FROM mcr.microsoft.com/dotnet/core/sdk:3.1 AS runtime-image
+ARG HOST_VERSION
+
+ENV PublishWithAspNetCoreTargetManifest=false
+
+RUN BUILD_NUMBER=$(echo ${HOST_VERSION} | cut -d'.' -f 3) && \
+    git clone --branch v${HOST_VERSION} https://github.com/Azure/azure-functions-host.git /src/azure-functions-host && \
+    cd /src/azure-functions-host && \
+    HOST_COMMIT=$(git rev-list -1 HEAD) && \
+    dotnet publish -v q /p:BuildNumber=$BUILD_NUMBER /p:CommitHash=$HOST_COMMIT src/WebJobs.Script.WebHost/WebJobs.Script.WebHost.csproj -c Release --output /azure-functions-host --runtime linux-x64 && \
+    rm -rf /azure-functions-host/workers/powershell/6/ && \
+    rm -rf /azure-functions-host/workers/powershell/7/runtimes/win* && \
+    mv /azure-functions-host/workers /workers && mkdir /azure-functions-host/workers && \
+    rm -rf /root/.local /root/.nuget /src
+
+RUN EXTENSION_BUNDLE_VERSION=1.6.0 && \
```

Comment:
Nit: This line and next line, we can make 1.6.0 to ARG. It might help visibility.

Reply:
Testing this now locally.  Will update in next commit